py/aou_workbench_client/data.py

```diff
@@ -0,0 +1,53 @@
+from aou_workbench_client.swagger_client.models.materialize_cohort_request import MaterializeCohortRequest
+from aou_workbench_client.swagger_client.models.table_query import TableQuery
+from aou_workbench_client.swagger_client.models.field_set import FieldSet
+from aou_workbench_client.swagger_client.models.column_filter import ColumnFilter
+from aou_workbench_client.swagger_client.models.operator import Operator
+from aou_workbench_client.swagger_client.models.result_filters import ResultFilters
+from aou_workbench_client.cohorts import materialize_cohort
+
+def load_data_table(cohort_name, table, columns=None, concept_ids=None,
```

Comment:
Could you document what types you are expecting?

Reply:
Done.